Gemfile

```diff
@@ -4,12 +4,13 @@ ruby "3.2.2"
 gem "defra_ruby_template",
     git: "https://github.com/DEFRA/defra-ruby-template",
     branch: "main"
+gem "faraday"
+gem "faraday-retry"
```

Comment:
I see we're adding faraday-retry here and in the back-office as well, but there's no config added for it. Are we happy with the default retry setting?

Reply:
I think it's used only for the changelog generator, so default should be fine.